internal/store/installation_backup.go

```diff
@@ -96,6 +97,51 @@ func (sqlStore *SQLStore) IsInstallationBackupRunning(installationID string) (bo
 	return ongoingBackups > 0, nil
 }
 
+// IsInstallationBackupBeingUsed checks if backup is being used by any DB restoration Operation or DB migration Operation
+func (sqlStore *SQLStore) IsInstallationBackupBeingUsed(backupID string) (bool, error) {
+	backupsCountBuilder := sq.
+		Select("Count (*)").
+		From(fmt.Sprintf("%s as b", backupTable)).
```

Comment:
Can you tell me why string formatting is being used here instead of the `?` syntax on the other lines?

Reply:
Those are used to format the table names with aliases, it is static element of the query not an external parameter therefore it cannot be formatted with `?`syntax.

Reply:
oh OK I understand, I guess the `AS` SQL keyword is new to me, thanks for clearing that up.